README.md

```diff
@@ -7,13 +7,15 @@
 <img src="https://weatherflow.com/wp-content/uploads/2016/05/Tempest-powered-by-01.svg" width="250">
 </p>
 
-Homebridge Plugin providing basic WeatherFlow Tempest support. Exposing 5 Acessories.
+Homebridge Plugin providing basic WeatherFlow Tempest support. Exposing 6 Acessories + Battery Sensor.
```

Comment:
This could just say since it lists them below.
```suggestion
Homebridge Plugin providing basic WeatherFlow Tempest support. Exposing 7 Accessories.
```